Assets/Scripts/WanderArea.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using UnityEngine;
+
+namespace Utils
```

Comment:
If you give it a namespace, then it should be in the corresponding folder :P
We should probably start using a lot of folders and namespaces anyways